woocommerce/woocommerce-ios
# SPE: Show "Add options" button in the bottom of product details

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8840

## Description

This PR adds "Add Options" button on the product form. It opens the same action sheet as "Add more details" button.
It also slightly updates appearance of a form and fixes dividers.

## Testing

1. Build and run the app in debug/alpha mode.
2. Open product details for any product (or start product creation flow).
3. Validate appearance of the form, check cell dividers.
4. Tap "Add Options".
5. Confirm action sheet with additional options is displayed.

## Screenshots

before|after
--|--
![Simulator Screen Shot - 1](https://user-images.githubusercontent.com/3132438/220067795-70c2aa65-061f-41fb-b280-716c5ddf83b9.png)|![Simulator Screen Shot - 2](https://user-images.githubusercontent.com/3132438/220067880-59f33673-8960-4577-86ad-460d3dfb0c0e.png)


---
